### ISSUE TITLE
Bug on sample: "Expandable+draggable with section" - it allows to drag section item between its children

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [h6ah4i/android-advancedrecyclerview#91]
This shouldn't be possible:

![device-2015-08-20-111616](https://cloud.githubusercontent.com/assets/5357526/9378790/ea50ca0e-472c-11e5-9cdb-3ee6087612bf.png)

**Comment by h6ah4i** [comment]
Hi. 
It is working as expcted, I think that behavior is not a bug. If you want to disallow dragging for group items, make the `onCheckGroupCanStartDrag()` method return `false` .

**Comment by AndroidDeveloperLB** [comment]
You don't understand. A is the one that has "a,b,c" children. It should be allowed to drag between B and C, but not in its own children. It's like A will be the child of itself.
Also, even when you stop the dragging, you will see that it won't allow you to really put A in between "a" and "b" .

**Comment by h6ah4i** [comment]
The behavior is by design of this library. Maybe, using the [ExpandableLayout](https://github.com/traex/ExpandableLayout) or some sort of custom views with RecyclerView will fits your expectation.

**Comment by AndroidDeveloperLB** [comment]
I only wanted to try out the sample. I consider this a bug because it lets you drag the parent into its children, but it won't let you put it there.

**Comment by wliu6v** [comment]
I'm agree with @AndroidDeveloperLB . I think the children should move with the parent when drag the parent. The parent & children should be a whole when start drag on the parent.

I don't know how to implement this effect, so I collapse all items when I start drag, and re-expand them when the drag ends.

**Comment by h6ah4i** [comment]
@AndroidDeveloperLB @wliu6v I also agree with your opinion that current behavior is weird and looks like something broken.

However, I'm not going to change the behavior. Because it is too difficult (and I think that is impossible...) to make treating a group and its children items as a single draggable object.

> I don't know how to implement this effect, so I collapse all items when I start drag, and re-expand them when the drag ends.

That is a good workaround for this issue. Maybe I can add a sample code.
